app/views/parents/student-progress/CampaignProgressView.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div
+    v-if="campaign"
+    class="campaign-progress"
+  >
+    <module-progress-data-component
+      :levels="sortedLevels"
+      :level-sessions="levelSessions"
+    />
```

Comment:
We're not passing the `language` here. (I have been religiously calling it `codeLanguage` to differentiate from `spokenLanguage` in new code, since it has gotten confusing in the past.) I suppose the language is specific to the LevelSession, so the code language property should initialize and exist at a lower level.